Tests/PostgresNIOTests/New/PSQLMetadataTests.swift

```diff
@@ -0,0 +1,17 @@
+import NIO
+import XCTest
+@testable import PostgresNIO
+
+class PSQLMetadataTests: XCTestCase {
+    func testSelect() {
+        XCTAssertEqual(100, PostgresQueryMetadata(string: "SELECT 100")?.rows)
+        XCTAssertEqual(0, PostgresQueryMetadata(string: "SELECT")?.rows)
```

Comment:
uff, this tests needs fixing as well!

```suggestion
        XCTAssertEqual(nil, PostgresQueryMetadata(string: "SELECT")?.rows)
```

Reply:
Tests fixed!